src/conways/point.rs

```diff
@@ -0,0 +1,31 @@
+use std::collections::HashSet;
+
+#[derive(PartialEq, Eq, Hash, Clone, Copy)]
+pub struct Point {
+    x_position: i32,
+    y_position: i32,
+}
+
+impl Point {
+    pub fn new(x_position: i32, y_position: i32) -> Self {
```

Comment:
I don't get what is the frame of reference for the creation of cells. Intuitively, I would think of the cell position as an xy coordinate in the first region (x and y both positive), but x and y are defined as `i32`, and when defining the cells you use both positive and negative coordinates. I don't understand the rationale. Is there something I am missing?

Reply:
With the implementation using ´HashSet´ the points could be anywhere in the xy plane, so it doesn't really matter if the positions are negative or positive

Reply:
I see. I think it's simpler to restrict the coordinates to a single quadrant. Don't worry, I'll close this one.